.github/workflows/prometheus-compliance-tests.yml

```diff
@@ -45,9 +45,10 @@ jobs:
         with:
           repository: prometheus/compliance
           path: compliance
```

Comment:
should we checkout a specific commit/tag?

Reply:
oh, I am not sure if we need to checkout specific tag. Since I see the tests are stable now at the main/recent commit.